extensions/meme_rater.py

```diff
@@ -0,0 +1,50 @@
+import os
+import hikari, lightbulb
+from PIL import Image
+import requests
+from snark import model
+from google.generativeai.generative_models import GenerativeModel
+
+plugin = lightbulb.Plugin("MemeRater")
+
+MEMES_CHANNEL_ID = os.environ["MEME_CHANNEL_ID"]
```

Comment:
os.environ.get instead to avoid potential KeyError?

also thinking at this point we should have an actual config file instead of random os.environ calls everywhere. Thoughts?

Reply:
+1